docker/builder/README.md

```diff
@@ -0,0 +1,99 @@
+Firefly Development Environment
+------------------------------- 
+
+A Docker based local development environment for Firefly.  This includes all runtime and development
+dependencies.  
+
+
+#### Dependencies
+
+ - Docker and Docker-Compose     
+    Starts here https://docs.docker.com/get-docker/
+ - docker-sync  
+    http://docker-sync.io/
+
+
+## Quick Start
+
+#### Create source repositories
+
+    mkdir cm && cd cm
```

Comment:
It should not matter.  It will be mounted into the container as `cm`, but should work fine if you name it something else on the host.  If not, I will fix.

Reply:
Funny, I actually rewrote the comment to say is cm required? Because I decided it was not.  Maybe I did not save my changes. You answer works either way but you might want to say that `cm` can be anything you want.